packages/fxa-auth-server/lib/signer.ts

```diff
@@ -2,15 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-'use strict';
+import { JWTool, PrivateJWK } from '@fxa/vendored/jwtool';
 
-const { JWTool } = require('@fxa/vendored/jwtool');
-
-module.exports = function (secretKeyFile, domain) {
-  const key = JWTool.JWK.fromFile(secretKeyFile, { iss: domain });
+module.exports = function (secretKeyFile: string, domain: string) {
+  const key = JWTool.JWK.fromFile(secretKeyFile, { iss: domain }) as PrivateJWK;
 
   return {
-    sign: async function (data) {
+    sign: async function (data: any) {
```

Comment:
I guess more of a nit, but is there anything more specific than `any` for this?
My <kbd>grep</kbd>-fu is failing me, as this was the only type information I can seemingly find for a random "profileChangedAt" prop:
```sh
git grep -Pn " profileChangedAt: .*;" | cat

libs/shared/db/mysql/account/src/lib/kysely-types.ts:77:  profileChangedAt: number | null; 
```

&hellip; but my unreliable guts are saying that the `Accounts` interface probably isn't a match at all:
https://github.com/mozilla/fxa/blob/c89279fac2c30886e0bccb8928d8755e6bd539fc/libs/shared/db/mysql/account/src/lib/kysely-types.ts#L60-L84

Reply:
Probably, but I was aiming for a minimal file conversion here so we can get a release out.